utils_cv/detection/bbox.py

```diff
@@ -112,6 +169,20 @@ def is_valid(self) -> bool:
             return False
         return True
 
+    def hflip(self, width) -> "_Bbox":
```

Comment:
Are those two functions used anywhere? If not then can remove.

Reply:
Removed